strapi/strapi
# [Improvement] Speed up Webpack 🎉🎉🎉

Signed-off-by: soupette <cyril@strapi.io>



### What does it do?

It uses [esbuild-loader](https://github.com/privatenumber/esbuild-loader) to load the files that don't require the @strapi/babel-plugin-switch-ee-ce plugin and replace the [terser-webpack-plugin](https://webpack.js.org/plugins/terser-webpack-plugin/) by the ESBuildMinifyPlugin one.


### Why is it needed?

The decrease the CPU usage when building the admin and to make the compilation super fast!

### How to test it?

The following commands should still work:
- `cd examples/getstarted && yarn build`
- `cd examples/getstarted && yarn develop --watch-admin`
- `cd packages/core/admin && yarn develop`

The build should not be broken.


### Related issue(s)/PR(s)

Fixes #12095
